pytest_catchlog.py

```diff
@@ -92,19 +97,91 @@ def pytest_addoption(parser):
                     help='disable printing caught logs on failed tests.')
     add_option_ini(parser,
                    '--log-format',
-                   dest='log_format', default=('%(filename)-25s %(lineno)4d'
-                                               ' %(levelname)-8s %(message)s'),
+                   dest='log_format',
+                   default=DEFAULT_LOG_FORMAT,
                    help='log format as used by the logging module.')
     add_option_ini(parser,
                    '--log-date-format',
-                   dest='log_date_format', default=None,
+                   dest='log_date_format',
+                   default=DEFAULT_DATE_FORMAT,
                    help='log date format as used by the logging module.')
+    add_option_ini(parser,
+                   '--log-extra-level',
+                   dest='log_extra_levels',
+                   type='args',
+                   action='append',
+                   help='extra logging levels that verbosity should be sensible to.')
 
 
 def pytest_configure(config):
     """Always register the log catcher plugin with py.test or tests can't
     find the  fixture function.
     """
+    verbosity = config.getoption('-v')
+    if verbosity < 1:
+        cli_handler_level = logging.FATAL
+    elif verbosity >= 1:
+        # Prepare the handled_levels dictionary
+        log_levels = []
+        available_levels = set([
+            #logging.FATAL,  # This will be the default console level if no -v is passed
```

Comment:
And still...

> why is the highest level not included at the first place? It would eliminate the need to start counting from 1 in the enumerate call below, and let us use zero-based indices I was talking about yesterday.